src/tools/genpolicy/tests/adapt_settings_for_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# usage: ./tests/adapt_settings_for_tests.sh
+
+jq '.request_defaults.CreateContainerRequest.allow_env_regex_map = {  
+  "JOB_COMPLETION_INDEX": "^[0-9]+$",
+  "CPU_LIMIT": "^[0-9]+$",
+  "MEMORY_LIMIT": "^[0-9]+$"
+}' genpolicy-settings.json > tmp-genpolicy-settings.json && mv tmp-genpolicy-settings.json genpolicy-settings.json
```

Comment:
@ms-mahuber jq ... genpolicy-settings.json | tee genpolicy-settings.json doesn't work for me. Looks like the file gets cleared instead of being edited in place as expected